Ryujinx/Ui/GameTableContextMenu.cs

```diff
@@ -586,5 +630,51 @@ private void ExtractLogo_Clicked(object sender, EventArgs args)
         {
             ExtractSection(NcaSectionType.Logo);
         }
+
+        private void OpenPtcDir_Clicked(object sender, EventArgs args)
+        {
+            string titleId   = _gameTableStore.GetValue(_rowIter, 2).ToString().Split("\n")[1].ToLower();
+            string ptcDir    = System.IO.Path.Combine(_virtualFileSystem.GetBasePath(), "games", titleId, "cache", "cpu");
+            
+            string mainPath   = System.IO.Path.Combine(ptcDir, "0");
+            string backupPath = System.IO.Path.Combine(ptcDir, "1");
+
+            if (!Directory.Exists(ptcDir))
+            {
+                Directory.CreateDirectory(ptcDir);
+                Directory.CreateDirectory(mainPath);
+                Directory.CreateDirectory(backupPath);
+            }
+            
+            Process.Start(new ProcessStartInfo
+            {
+                FileName        = ptcDir,
+                UseShellExecute = true,
+                Verb            = "open"
+            });
+        }
+        
+        private void PurgePtcCache_Clicked(object sender, EventArgs args)
+        {
+            string titleId         = _gameTableStore.GetValue(_rowIter, 2).ToString().Split("\n")[1].ToLower();
+            string cacheFileName   = _gameTableStore.GetValue(_rowIter, 4) + ".cache";
```

Comment:
```suggestion
            string cacheFileName = _gameTableStore.GetValue(_rowIter, 4) + ".cache";
```
And align the titleId and the cachePath assign too ;)!